foundationdb/README.md

```diff
@@ -0,0 +1,149 @@
+# Agent Check: FoundationDB
+
+## Overview
+
+This check monitors [FoundationDB][1] through the Datadog Agent. Aside from
+checking the FoundationDB cluster is healthy, it also collects numerous metrics
+and, optionally, FoundationDB transaction logs.
+
+## Setup
+
+Both the check and the metrics apply to the FoundationDB cluster as a whole,
+and so should only be installed on one host (which need not be one that is
+running FoundationDB, but just one with access to it). The host chosen to
+collect the metrics is required to have the [FoundationDB client][8]
+installed. It is possible to monitor multiple FoundationDB clusters if
+desired, by adding multiple instances in the configuration file.
+
+By contrast, logs are written on each machine, and thus for collecting logs it
```

Comment:
```suggestion
Since logs are written on each machine, to collect logs, configure configure FoundationDB log collection on every host.
```

Reply:
Duplicated word "configure" in the correction

Reply:
Applied manually with the dupe word removed.